node/Cargo.toml

```diff
@@ -1,13 +1,13 @@
 [package]
 name = "data-avail"
 version = "1.5.0"
-description = "Polygon Data Avilability Node."
+description = "Data Avilability Node"
```

Comment:
Should we change it to `Avail Node`? 
minor typo: `Avilability` -> `Availability`